src/utils/misc.ts

```diff
@@ -153,3 +154,25 @@ export const getUpdateAuthor: (id: string, isReviewer: boolean) => UpdateAuthor
   id,
   role: isReviewer ? DacoRole.ADMIN : DacoRole.SUBMITTER,
 });
+
+export const appHistoryTSVColumns: ColumnHeader[] = [
+  { name: 'Application #', accessor: 'appId' },
+  {
+    name: 'Date of Status Change',
+    accessor: 'date',
+    format: (value: string) => moment(value).format('YYYY-MM-DD'),
+  },
+  { name: 'Application Status', accessor: 'eventType' },
+  { name: 'Application Type', accessor: 'appType' },
+  { name: 'Action Performed By', accessor: 'role' },
+  { name: 'Days Since Last Status Change', accessor: 'daysElapsed' },
+  { name: 'Institution', accessor: 'institution' },
+  { name: 'Country', accessor: 'country' },
+  { name: 'Applicant', accessor: 'applicant' },
+  { name: 'Project Title', accessor: 'projectTitle' },
+  { name: 'Ethics Letter', accessor: 'ethicsLetterRequired' },
+];
```

Comment:
this could be wrapped in the service file as well, is there a reason I may be missing to have it located here ?

Reply:
nope - actually now that i think about it i might just put this in the tsv function itself, it doesn't need to be accessed by anything else